charts/bdrs-server/templates/deployment.yaml

```
@@ -99,6 +99,7 @@ spec:
           image: "tractusx/bdrs-server:{{ .Values.server.image.tag | default .Chart.AppVersion }}"
           {{- end }}
 
+          args: [ --log-level={{ .Values.server.logs.level | required ".Values.server.logs.level is required" }} ]
```

nit: should never be an issue, but is consistent with others. same for the other.
```suggestion
          args: [ --log-level={{ .Values.server.logs.level | required ".Values.server.logs.level is required" | quote  }} ]
```